app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,24 @@
+class TasksController < ApplicationController
+  def daily
+    # Hardcode first child for Phase 1
+    @child = Child.first || Child.create!(name: "Test Child")
+    @date = params[:date] ? Date.parse(params[:date]) : Date.current
+    @tasks = @child.active_tasks.ordered
+    @is_today = @date == Date.current
+  end
+
+  def toggle
```

Comment:
This should be in the task_completions controller and should follow restful conventions.